src/sbsign/sbsign.c

```diff
@@ -67,6 +67,11 @@ static int help(int argc, char *argv[], void *userdata) {
                "     --private-key-source=file|provider:PROVIDER|engine:ENGINE\n"
                "                         Specify how to use KEY for --private-key=. Allows\n"
                "                         an OpenSSL engine/provider to be used for signing\n"
+               "     --prepare-offline-signing\n"
+               "                         Write the data that should be signed instead of the signed data"
+               "     --signed-data=PATH  Path to the data that was signed offline\n"
+               "     --signed-data-signature=PATH\n"
+               "                         Path to the signature of the data that was signed offline\n"
```

Comment:
would be good to say the format here, i.e. pkcswateverthatwas in PEM format

Reply:
The thing is this doesn't really have a format, it's just a blob of data that needs to be signed, but I can clarify that I guess.

Reply:
the --signed-data= is just a blob, but the signature data has a format no?

if not, say "raw"